src/generate.js

```diff
@@ -40,24 +40,44 @@ function cloneRepoApi(repoName) {
   return repoDirectory;
 }
 
+// run npm install on any specs in the dependencies section of the repo's package.json
+function loadOutsideSpecs(repoDirectory) {
+  console.log('Loading swagger specs from outside projects...');
+
+  // get package.json "dependencies" section
+  const dependencyObj = JSON.parse(fs.readFileSync(`${repoDirectory}/package.json`)).dependencies;
+
+  // parse into an array of { name, version } objects
+  // then filter to spec packages only
+  const specNameRegExp = new RegExp('^@gasbuddy/.+-spec$');
+  const specs = Object.keys(dependencyObj)
+    .map(depName => ({ name: depName, version: dependencyObj[depName] }))
+    .filter(dep => specNameRegExp.test(dep.name));
+
+  for (const spec of specs) {
+    childProcess.execSync(`cd ${repoDirectory} && npm install ${spec.name}@${spec.version}`);
```

Comment:
does just running npm install with no package specified work as well? If so it seems like a much simpler more universal solution even if it takes a little bit longer to run.